man/dockerd.8.md

```diff
@@ -266,7 +267,10 @@ output otherwise.
   Default is false.
 
 **--userland-proxy**=*true*|*false*
-    Rely on a userland proxy implementation for inter-container and outside-to-container loopback communications. Default is true.
+  Rely on a userland proxy implementation for inter-container and outside-to-container loopback communications. Default is true.
+
+**--userland-proxy-path**=""
+  Set the userland proxy binary path instead of relying on `docker-proxy` to be present in `$PATH`.
```

Comment:
"Specify the full filename of the userland proxy binary, instead of looking for `docker-proxy` in the `$PATH`"